### ISSUE TITLE
Hovering Over % Values Calculated in Custom SQL Metrics is showing the row value- 16.25 is shown as .1625 instead if 16.25

### ISSUE THREAD

**Issue by hk-the-analyst** [apache/superset#28140]
![image](https://github.com/apache/superset/assets/123436992/9620d113-cedb-4e68-921c-581cb06248bf)

![image](https://github.com/apache/superset/assets/123436992/bee49a32-5bb3-4c4c-a3b2-c6f6b07bf19c)

**Comment by rusackas** [comment]
@dosu-bot

**Comment by rusackas** [comment]
I thought this is the desired effect... when you hover, you get more detail (the exact value) rather than the formatted one. This applies to percentages, or other formats (e.g. 12,002 displayed as 12K). 

Should be simple enough to have the "title" attribute of the HTML cell show the formatted value, but I always figured the title (hover) attribute with the _actual_ value was pretty handy. 

CC @mistercrunch for any historical context here, or @kasiazjc for any design thoughts.

**Comment by msyavuz** [comment]
I agree with @rusackas here that this seems like the desired effect of the tooltip

**Comment by rusackas** [comment]
Closing this for now... we can open a new Issue or a Discussion if this persists.
